lib/ddtrace/contrib/action_cable/ext.rb

```diff
@@ -4,7 +4,8 @@ module ActionCable
       # ActionCable integration constants
       module Ext
         APP = 'action_cable'.freeze
-        ENV_ANALYTICS_ENABLED = 'DD_ACTION_CABLE_ANALYTICS_ENABLED'.freeze
+        ENV_ANALYTICS_ENABLED = 'DD_TRACE_ACTION_CABLE_ANALYTICS_ENABLED'.freeze
+        ENV_ANALYTICS_ENABLED_OLD = 'DD_ACTION_CABLE_ANALYTICS_ENABLED'.freeze
         ENV_ANALYTICS_SAMPLE_RATE = 'DD_ACTION_CABLE_ANALYTICS_SAMPLE_RATE'.freeze
```

Comment:
These need to change as well, `DD_TRACE_<INTEGRATION>_ANALYTICS_SAMPLE_RATE`, sorry if the spec wasn't clear about that

Reply:
gotcha, updated